docs/source/changelog.rst

```diff
@@ -1,6 +1,117 @@
 Changelog
 =========
 
+2020.12.0
+---------
+
+Released on December 10, 2020
+
+Highlights
+^^^^^^^^^^
+
+- The scheduler can now receives Dask ``HighLevelGraph`` s instead of raw dictionary task graphs.
+  This allows for a much more efficient communication of task graphs from the client to the scheduler.
```

Comment:
I've always used four spaces for indents after a bullet point.  This might not be necessary though.  It might be worth checking.

Reply:
Checked and two spaces is rendering as expected

![Screen Shot 2020-12-10 at 5 12 20 PM](https://user-images.githubusercontent.com/11656932/101841150-02a55200-3b0b-11eb-9f9b-0924ee4ceaab.png)